servo/telemetry.py

```diff
@@ -1,9 +1,43 @@
+from __future__ import annotations
+
+import abc
 import os
 import platform
 from typing import Dict
 import pydantic
+import enum
+import json
+import logging
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import asyncio
+import aiofiles
+import backoff
+import devtools
+import httpx
+import pydantic
+from semver import b
 
 import servo
+import servo.api
+from servo.logging import logs_path
+from servo.types import JSON_FORMAT
+
+
+ONE_MiB = 1048576
+DIAGNOSTICS_MAX_RETRIES = 20
+
+# Intercept backoff decorator logs
+logging.getLogger('diagnostics-backoff').setLevel(logging.ERROR)
```

Comment:
The [backoff decorator](https://github.com/litl/backoff#logging-configuration) logs to `INFO` by default and I don't want the logs to crowd with retry attempts for diagnostics at the info level

Reply:
Good thinking. Only issue is you're increasing the verbosity instead of decreasing it: https://loguru.readthedocs.io/en/stable/api/logger.html#levels

I suggest setting to the DEBUG level (also are you sure its getting the same sinks as the main logging assembly so that it gets logged to stdout and the servo.log file?)

Reply:
Good catch on adding the InterceptHandler. As for the level, they actually say "if you would instead like to log only when a giveup event occurs, set the logger level to ERROR." Tested to verify~

Reply:
Okay thanks for clarifying, I misunderstood how `setLevel` was working; I thought it was setting the log level of logs emitted by backoff but its actually configuring the minimum severity threshold for backoff's logging filter :+1: